schema/aggregations/filter_artworks_aggregation.js

```diff
@@ -24,17 +24,11 @@ export const ArtworksAggregation = new GraphQLEnumType({
     MEDIUM: {
       value: 'medium',
     },
-    FOR_SALE: {
```

Comment:
This doesn't work currently here...the format of this aggregation is slightly different than others in the API response.

Here's how metaphysics handles it:

<img width="817" alt="screen shot 2016-02-11 at 2 25 05 pm" src="https://cloud.githubusercontent.com/assets/1457859/12987702/3f6fb69e-d0cb-11e5-8594-0084db12b426.png">

:(

I removed it until we fix the count parsing here.